spec-0010/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "SPEC 10 — Changelog"
+number: 10
+date: 2024-06-03
+author:
+  - "Inessa Pawson <inessapawson@gmail.com>"
+discussion: https://discuss.scientific-python.org/t/
+endorsed-by:
+---
+
+## Description
+
+SPEC 10 provides guidelines and best practices for maintaining a changelog file
+for libraries in the Scientific Python ecosystem.
+
+### Core Project Endorsement
+
+<!--
+Briefly discuss what it means for a core project to endorse this SPEC.
+-->
+
+### Ecosystem Adoption
+
+The endorsement of this SPEC signifies your project's support for the guidelines 
+laid out in the document.
+
+## Implementation
+
+Keeping a clear, organized, and human-readable record of notable changes for each 
+version of a Python library is essential for the project’s maintenance and sustainability. 
+It promotes transparency and trust between the project’s leadership team, developers, and users.
+
+### What Is a Changelog?
+
+A changelog is a file which contains a curated, chronologically ordered list of 
```

Comment:
Maybe remove the "chronologically" part? I think more useful is ordering by significance and / or grouping by category.

Reply:
> Maybe remove the "chronologically" part? I think more useful is ordering by significance and / or grouping by category.

💯 on structuring by logic rather than time